doc/classes/RigidBody.xml

```diff
@@ -5,10 +5,9 @@
 	</brief_description>
 	<description>
 		This is the node that implements full 3D physics. This means that you do not control a RigidBody directly. Instead you can apply forces to it (gravity, impulses, etc.), and the physics simulation will calculate the resulting movement, collision, bouncing, rotating, etc.
-		This node can use custom force integration, for writing complex physics motion behavior per node.
-		This node can shift state between regular Rigid body, Kinematic, Character or Static.
-		Character mode forbids this node from being rotated.
-		As a warning, don't change RigidBody's position every frame or very often. Sporadic changes work fine, but physics runs at a different granularity (fixed hz) than usual rendering (process callback) and maybe even in a separate thread, so changing this from a process loop will yield strange behavior.
+		A RigidBody has 4 behavior [member mode]s: Rigid body, Kinematic, Character or Static.
```

Comment:
I would drop "body" since you don't use it on the other 3 mode names.
Also, please add an Oxford comma after "Character".